sched/Kconfig

```diff
@@ -351,7 +351,6 @@ config SMP
 	depends on ARCH_HAVE_TESTSET
 	depends on ARCH_INTERRUPTSTACK != 0
 	select SPINLOCK
-	select SCHED_RESUMESCHEDULER
 	select IRQCOUNT
```

Comment:
@hujun260 
We need clearer explanation in the commit log why you remove this line.